### PR TITLE
Mock css modules with cssTransform

### PR DESCRIPTION
**Pull request by morgs32**

### CODE REVIEW

File: packages/react-scripts/config/jest/cssTransform.js [hunk 1]
```diff
@@ -11,13 +11,44 @@
...
+
+module.exports = idObj;
+
+    `
   },
   getCacheKey() {
     // The output is always the same.
-    return 'cssTransform';
+    return 'cssTradform';
   },
```
Comment (wilsonyhlee) [inline packages/react-scripts/config/jest/cssTransform.js]: @morgs32 is this intentional?

**Review by wilsonyhlee — DISMISSED** [review]
Left a comment (not blocking). Let's merge.

File: packages/react-scripts/config/jest/cssTransform.js (hunk 1 above)
Comment (morgs32) [reply packages/react-scripts/config/jest/cssTransform.js]: Nope sure isn't thanks for catching.

**Review by wilsonyhlee — APPROVED** [review]
Looks look. Let's merge!

**Pull request merged**